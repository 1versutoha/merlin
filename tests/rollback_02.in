["reset","name","test.ml"]
["tell","source","let f = ignore\n"]
["tell","end"]
["seek","exact",{"line":1,"col":0}]
["errors"]
["seek","before",{"line":2,"col":0}]
["reset","name","test.ml"]
["tell","source","let f = ignore\n\n5\n"]
["tell","end"]
["seek","exact",{"line":3,"col":0}]
["errors"]
["seek","before",{"line":2,"col":0}]
["reset","name","test.ml"]
["tell","source","let f = ignore\n;;\n5\n"]
["tell","end"]
["seek","exact",{"line":3,"col":0}]
["errors"]
["seek","before",{"line":4,"col":0}]
["seek","exact",{"line":1,"col":4}]
["type","enclosing",{"expr":"f","offset":0},{"line":1,"col":4}]
["seek","before",{"line":2,"col":0}]
["reset","name","test.ml"]
["tell","source","let f = ignore\n;\n5\n"]
["tell","end"]
["seek","exact",{"line":3,"col":0}]
["errors"]
["seek","before",{"line":4,"col":0}]
["seek","exact",{"line":1,"col":4}]
["type","enclosing",{"expr":"f","offset":0},{"line":1,"col":4}]
