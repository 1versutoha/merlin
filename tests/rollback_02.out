["return",{"line":1,"col":0}]
["return",null]
["return",{"line":1,"col":14}]
["return",{"line":1,"col":14}]
["return",[]]
["return",{"line":1,"col":14}]
["return",{"line":1,"col":0}]
["return",null]
["return",{"line":3,"col":1}]
["return",{"line":3,"col":1}]
["return",[]]
["return",{"line":1,"col":0}]
["return",{"line":1,"col":0}]
["return",null]
["return",{"line":3,"col":1}]
["return",{"line":3,"col":1}]
["return",[]]
["return",{"line":3,"col":1}]
["return",{"line":1,"col":14}]
["return",[{"start":{"line":1,"col":4},"end":{"line":1,"col":5},"type":"'a -> unit"},{"start":{"line":1,"col":4},"end":{"line":1,"col":5},"type":"'a -> unit"}]]
["return",{"line":1,"col":14}]
["return",{"line":1,"col":0}]
["return",null]
["return",{"line":3,"col":1}]
["return",{"line":3,"col":1}]
["return",[{"type":"warning","start":{"line":1,"col":8},"end":{"line":1,"col":14},"valid":true,"message":"Warning 5: this function application is partial,\nmaybe some arguments are missing.\n"}]]
["return",{"line":3,"col":1}]
["return",{"line":3,"col":1}]
["return",[{"start":{"line":1,"col":4},"end":{"line":1,"col":5},"type":"int"},{"start":{"line":1,"col":4},"end":{"line":1,"col":5},"type":"int"}]]
